src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -153,6 +155,10 @@ namespace
             p_default, FALSE,
             p_ui, ParamMapIdVisibility, TYPE_SINGLECHECKBOX, IDC_BUTTON_PHOTON_TARGET,
         p_end,
+        ParamIdShadowTerminatorCorrection, L"shadow_terminator_correction", TYPE_FLOAT, P_TRANSIENT, 0,
+            p_ui, ParamMapIdVisibility, TYPE_SPINNER, EDITTYPE_FLOAT, IDS_TEXT_SHADOW_TERMINATOR_CORRECTION, IDS_SPINNER_SHADOW_TERMINATOR_CORRECTION, SPIN_AUTOSCALE,
+            p_default, 0.0f, p_range, 0.0f, 0.5f,
```

Comment:
Are these soft or hard bounds?

Reply:
Hard bounds. Max does not have the soft bounds like Maya has. I have seen that also in appleseed.studio the upper limit has been set to 0.5f (hard limit).